app/javascript/alchemy_admin/utils/dom_helpers.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * create a HTML element
+ * @param {string} text
+ * @returns {HTMLElement}
+ */
+export function createHtmlElement(text) {
+  const element = document.createElement("div")
```

Comment:
It seems to be common sense to use a `template` element for that. 

https://youmightnotneedjquery.com/#create_elements